src/components/Email/Email.tsx

```diff
@@ -0,0 +1,74 @@
+import React, {useState} from 'react'
+import './Email.scss'
+import {Form} from 'react-final-form'
+import MessageEmail from '../MessageEmail/MessageEmail'
+import PasswordEmail from '../PasswordEmail/PasswordEmail'
+import YourEmail from '../YourEmail/YourEmail'
+
+interface IProps {
+    component: boolean;
+    openComponent: any
+}
+
+const Email:React.FC<IProps> = ({component, openComponent}) => {
```

Comment:
Name of this component is a bit confusing, because it's essentially some kind of login (create account or continue as guest). Consider some more verbose naming